cratedb_toolkit/io/cli.py

```
@@ -67,3 +67,60 @@
+    """
+    Export data from CrateDB.
+    """
+    return boot_click(ctx, verbose, debug)
+
+
+@make_command(cli_save, name="table")
+@click.argument("url")
+@option_cluster_id
+@option_cluster_name
+@option_cluster_url
+@click.option("--schema", envvar="CRATEDB_SCHEMA", type=str, required=False, help="Schema where to import the data")
```

Pylint (3.3.7)

[convention] Line too long (116/100)

(C0301)